src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -1,7 +1,77 @@
+// Dependencies
 import React from 'react';
+import PropTypes from 'prop-types';
+import SortableTable from '@department-of-veterans-affairs/formation-react/SortableTable';
+import isEmpty from 'lodash/isEmpty';
+import { connect } from 'react-redux';
 
-function SearchResults() {
-  return <div />;
-}
+const tableFields = [
+  {
+    label: 'VA form number',
+    value: 'tableFieldID',
+  },
+  {
+    label: 'Form name',
+    value: 'tableFieldFormName',
+  },
+  {
+    label: 'Description',
+    value: 'tableFieldDescription',
+  },
+  {
+    label: 'Available Online',
+    value: 'tableFieldAvailableOnline',
+  },
+];
 
-export default SearchResults;
+const SearchResults = ({ query, results }) => {
+  // Render null if there are no results.
+  if (isEmpty(results)) {
+    return null;
+  }
+
+  return (
+    <div className="find-va-forms-results vads-u-display--flex">
+      <h2 className="vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal">
+        Showing results for "<strong>{query}</strong>"
+      </h2>
+
+      <SortableTable
+        className="va-table"
+        currentSort={{ order: 'ASC', value: 'title' }}
+        data={results}
+        fields={tableFields}
```

Comment:
So I guess we need to add the `onSort` or `onHeaderClick` prop to this component in order to enable column sorting. I'm guessing this PR is a dependency, https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/191. I think we can leverage `onSort` for now and update later.

Reply:
Writing a follow-up ticket for this...

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/3882